bower.json

```diff
@@ -1,29 +1,53 @@
 {
   "name": "device-orientation",
-  "description": "Polymer element to detecting device orientation",
+  "description": "Polymer element to detecting device orientation and motion",
   "authors": [
     "Abdón Rodríguez Davila <a@abdonrd.com>"
   ],
   "keywords": [
     "web-component",
     "polymer",
     "device",
+    "position",
     "orientation",
+    "motion",
     "sensor"
   ],
   "license": "https://abdonrd.github.io/LICENSE.txt",
   "repository": {
     "type": "git",
     "url": "git://github.com/abdonrd/device-orientation.git"
   },
-  "main": "device-orientation.html",
+  "main": [
+    "device-motion.html",
+    "device-orientation.html"
+  ],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.6.0"
+    "polymer": "Polymer/polymer#1 - 2"
```

Comment:
Please, keep the dependencies.

I think it's best if you don't update things external to the original PR. In this case, only add `DeviceMotionEvent` support.